font/ttf.ksy

```diff
@@ -242,6 +242,112 @@ types:
               - id: repeat_value
                 type: u1
                 if: repeat
+  gvar:
+    doc: >
+      gvar - Glyph Variations Table
+      This table provides all of the variation data that describe how TrueType glyph outlines in a 'glyf' table change across the font’s variation space.
+    seq:
+      - id: major_version
+        type: u2
+      - id: minor_version
+        type: u2
+      - id: axis_count
+        type: u2
+      - id: shared_tuple_count
+        type: u2
+      - id: shared_tuples_offset
+        type: u4
+      - id: glyph_count
+        type: u2
+      - id: flags
+        type: u2
+      - id: glyph_variation_data_array_offset
+        type: u4
+      - id: glyph_variation_data_offsets
+        type:
+          switch-on: flags & 0x01
```

Comment:
`flags` should be a a type of `b1`

Reply:
flags is defined as a uint16, are you requesting an array of b1? This would actually look rather strange to anyone familiar with the format I think, since this is actually a big-endian value and the specification consistently refers to flag masks by value.

Reply:
I changed the flags to a set of reserved flags and named this flag, removing all of the bit masking in the process.